### PR TITLE
add condition to only use env creds for wcs nlu

### PR DESCRIPTION
**Pull request by neilsbhargava**
A PR for a bug discovered by investigating [this](https://github.ibm.com/ConsumerIoT/WA-dev-issues/issues/852) issue regarding regex NLU engine failing to initiate when the skill is started in "wcs" and "regexp" mode with a corresponding .env file containing the wcs credentials.

### CODE REVIEW

**Comment by erezbi** [comment]
We have to get this tested before merging it.
Also, changed the PR to merge into 'pre-release' instead of 'master'.

File: lib/nlu/factory.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ function getNLU(type, name) {
             let nlu;
             if(process.env.WCS_USERNAME && process.env.WCS_URL && process.env.WCS_PASSWORD && process.env.WCS_VERSION_DATE
                 && process.env.WCS_VERSION && process.env.WCS_WORKSPACE_ID && process.env.WCS_WORKSPACE_NAME &&
-                process.env.WCS_WORKSPACE_LANGUAGE) {
+                process.env.WCS_WORKSPACE_LANGUAGE && type == 'wcs') {
```
Comment (offerakrabi) [inline lib/nlu/factory.js]: I think it's better to use === rather then ==

Comment (neilsbhargava) [reply lib/nlu/factory.js]: Yeah, thats a good idea.

**Comment by neilsbhargava** [comment]
Anything else required from my side?

**Comment by troyibm** [comment]
@erezbi is this going to get into the 11-01 release??

**Comment by offerakrabi** [comment]
@erezbi we can close this, added this fix to the wa api key PR

**Pull request closed without merging**